server/commands/VolumeCommand.ts

```diff
@@ -2,17 +2,40 @@ import CommandDefinition from '../domain/CommandDefinition';
 import IoConnection from '../clients/IoConnection';
 import VolumeEventMessage from "../domain/event-messages/VolumeEventMessage";
 
+const volChangeTypes: { [s: string]: string; } = {
+  '+': 'increase',
+  '-': 'decrease',
+  default: 'set',
```

Comment:
Both this object and `changeType` field inside `VolumeEventMessage` should be the same enum type.
https://www.typescriptlang.org/docs/handbook/enums.html